src/common/sourceUtils.ts

```diff
@@ -241,7 +241,7 @@ export async function checkContentHash(
   }
 
   if (!contentHash) {
-    const exists = await fsUtils.exists(absolutePath);
+    const exists = fs.existsSync(absolutePath);
```

Comment:
why was this change made? We don't want to block the main thread of FS operations if it can be helped

Reply:
That was my mistake. Good catch!